Additions/UIView-KIFAdditions.m

```diff
@@ -485,6 +485,8 @@ - (void)dragPointsAlongPaths:(NSArray *)arrayOfPaths {
             }
             UIEvent *eventDown = [self eventWithTouches:[NSArray arrayWithArray:touches]];
             [[UIApplication sharedApplication] sendEvent:eventDown];
+            
+            CFRunLoopRunInMode(UIApplicationCurrentRunMode, DRAG_TOUCH_DELAY, false);
```

Comment:
In your comments for this pull request you talk about setting the event timestamp, which makes perfect sense. On the other hand you do not mention pumping the run loop. Can you explain what this line is for?